pnpm-workspace.yaml

```diff
@@ -9,6 +9,7 @@ packages:
   - 'packages/dashboard'
   - 'packages/mask'
   - 'packages/injected-script'
+  - 'packages/gun-utils'
```

Comment:
why don't we use glob pattern here?

```yml
packages:
  - 'packages/*'
  - 'packages/polyfills/*'
 ```

Reply:
More explicit is good? 🤔

Reply:
I think we should use glob pattern instead, there are several places we need to update manually  when we add a new plugin, including but might not limited to:

- pnpm-workspace.yaml
- webpack/config.ts
- .i18n-codegen.json
- plugin-infra/register.ts
- packages/plugin-infra/src/types.ts

We can save steps and reduce possible mistakes when add a new plugin.

Reply:
I have opened a new issue for it. https://github.com/DimensionDev/Maskbook/issues/5666